src/renderer/components/DependsOnView.tsx

```diff
@@ -412,6 +412,96 @@ const DependsOnView: React.FC<Props> = ({
     // only fire when options.ports changes
   }, [options.ports]);
 
+  /**
+   *********************
+   * VOLUMES OPTION TOGGLE
+   *********************
+   */
+  useEffect(() => {
+    // VOLUMES LOCATION
+    const x = 8;
+    const y = 20;
+    const width = 10;
+    const height = 10;
+    // VOLUMES VARIABLES
+    let nodesWithVolumes: d3.Selection<SVGGElement, SNode, any, any>;
+    const volumes: d3.Selection<SVGRectElement, SNode, any, any>[] = [];
+    const volumeText: d3.Selection<SVGTextElement, SNode, any, any>[] = [];
+    if (options.volumes) {
+      // select all nodes with volumes
+      nodesWithVolumes = d3
+        .select('.nodes')
+        .selectAll<SVGGElement, SNode>('g')
+        .filter((d: SNode) => d.volumes.length > 0);
+
+      // iterate through all nodes with volumes
+      nodesWithVolumes.each(function(d: SNode) {
+        const node = this;
+        // iterate through all volumes of node
+        d.volumes.forEach((vString, i) => {
+          let onClick = false;
+          let onceClicked = false;
+          // add svg volume
+          const volume = d3
+            .select<SVGElement, SNode>(node)
+            .append('rect')
+            .attr('class', 'volumeSVG')
+            .attr('fill', () => {
+              let slicedVString: string = colorSchemeHash(
+                vString.slice(0, vString.indexOf(':')),
+              );
+              return slicedVString;
+            })
+            .attr('width', width)
+            .attr('height', height)
+            .attr('x', x)
+            .attr('y', y + i * 12)
+            .on('mouseover', () => {
+              return vText.style('visibility', 'visible');
+            })
+            .on('mouseout', () => {
+              !onClick
+                ? vText.style('visibility', 'hidden')
```

Comment:
can this be handled with class and styled within sass file?

Reply:
not sure how to do this, if changing the class name styles wouldn't all the text node change and not that individual text node?

Reply:
can you just append a string to the classname that would process as hidden?